schema/system/scalars/index.ts

```diff
@@ -1,16 +1,18 @@
 import type { GraphQLScalarType } from "graphql";
 import {
-  GraphQLDuration,
+  // GraphQLDuration,
```

Comment:
are we leaving this for a later date?

Reply:
Potentially yeah. It's a different format (ISO Duration) than what we do for tat right now (seconds) so can't use it